src/status_im2/subs/shell.cljs

```diff
@@ -97,12 +100,9 @@
    (community-card community community-id)
    {:content  {:community-channel {:emoji        (:emoji channel)
                                    :channel-name (str "# " (:name channel))}}
-    :on-press (fn []
-                (re-frame/dispatch [:navigate-to :community-overview community-id])
-                (js/setTimeout
-                 #(re-frame/dispatch [:chat/navigate-to-chat channel-id])
-                 100))}))
+    :on-press #(re-frame/dispatch [:chat/navigate-to-chat channel-id])}))
```

Comment:
doesn't look right, to have handlers in subscriptions

Reply:
Hi, thank you for reviewing PR. Addressed in https://github.com/status-im/status-mobile/pull/15893/commits/97acd368c45e17d183248c474d9fa86a5ed88cd5